### PR TITLE
Update referrer-policy with bug links

### PR DESCRIPTION
**Pull request by zbjornson**
(Is there a way to repeat a note so that the footnote number is the same in MDN?)

### CODE REVIEW

File: http/headers/referrer-policy.json [hunk 1]
```diff
@@ -155,7 +157,8 @@
         "support": false
       },
       "Chrome": {
-        "support": false
+        "support": false,
+        "notes": ["See <a href=\"https://bugs.chromium.org/p/chromium/issues/detail?id=627968\">Chromium bug 627968</a> and <a href=\"https://bugs.webkit.org/show_bug.cgi?id=154588\">WebKit bug 154588</a>."]
```
Comment (Elchi3) [inline http/headers/referrer-policy.json]: Why the WebKit bug here? WebKit is not used in Chrome anymore.

**Comment by Elchi3** [comment]
> (Is there a way to repeat a note so that the footnote number is the same in MDN?)

Currently, MDN does that automatically if the notes are identical. See e.g. https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control#Browser_compatibility

**Comment by zbjornson** [comment]
Sorry for delay, missed your comment.

**Comment by Elchi3** [comment]
Sorry for the delay and thanks for your contribution!

**Pull request merged**